### DISCUSSION TITLE
Login e sezione commenti

### DISCUSSION THREAD

**Discussion by gabacode** [opendatasicilia/opendatasicilia.it#36] [Domande]
Al momento sul portale WordPress le registrazioni sono disabilitate e l'accesso è esclusivo ai membri di tipologia Autore, Editore, e Amministratore.

In riferimento alla nota n.3 su https://xd.adobe.com/view/c9fc700a-2a14-4765-b529-af35e93fe73b-9caf/

Come vogliamo gestire le registrazioni, e il login, nel futuro portale? Conviene tenere un link che porti al wp-admin nel menu principale? Stessa cosa per quanto riguarda i commenti, gli utenti registrati al momento possono commentare se loggati, o tramite Disqus (mi sembra di capire).

Alcuni possibili scenari:
1. Registrazioni chiuse, commenti tramite disqus o amministratori loggati (situazione attuale). Avendo in futuro il frontend "staccato" dal backend, il miglior modo per commentare da parte degli admin sarebbe tramite il pannello wp-admin.
2. Si crea una sezione "Profilo" e Login all'interno del frontend, gli utenti loggati mandano il commento tramite il frontend, in un paio di minuti spunta il commento. (i commenti vanno approvati?)
3. Commenti solo su disqus (istantanei) e login per gli admin "celato" via bookmark ma non presente sul menu.
4. Registrazioni aperte a tutti come normali utenti, e login per amministratori tramite wp-admin
5. Altre idee?

**Comment by gbvitrano** [comment]
Registrazione solo per autori, l'utente normale non ha motivo di registarsi e può commentare tramite  Disqus.

Per la posizione del login sono d'accordo ad inserila nel menù del footer, se non addittura occultarla e comunicare l'indirizzo del login solo ai vari autotri. Io nel mio WP ho cambiato/modificato il percorso tradizionale.

Una cosa ancora non ho capito, il template che ho installato in WP è utile? il sito può essere già pubblicato temporaneamente online ora nell'attesa della versione definitiva di grafica e template? 
Ha senso che ci continui a giocare o a breve ci sarà la nuova versione?

![image](https://user-images.githubusercontent.com/20488693/145690353-08965446-7626-4c87-8461-0afe6215bab9.png)

**Reply by gabacode** [reply]
Grazie per la risposta!

> inserila nel menù del footer, se non addittura occultarla e comunicare l'indirizzo del login solo ai vari autotri. Io nel mio WP ho cambiato/modificato il percorso tradizionale.

Effettivamente sarebbe un livello di sicurezza in piu, la maggior parte dei bot punta tantissimo a wp-admin.php (persino su [infocomuni.eu](https://infocomuni.eu) 😅)

> Registrazione solo per autori, l'utente normale non ha motivo di registarsi e può commentare tramite Disqus.

Mi sembra la soluzione piu pulita, semplice e sicura. Per permettere agli utenti di rimanere in contatto facciamo un form per la mailing-list?

> Una cosa ancora non ho capito, il template che ho installato in WP è utile? il sito può essere già pubblicato temporaneamente online ora nell'attesa della versione definitiva di grafica e template? Ha senso che ci continui a giocare o a breve ci sarà la nuova versione?

E' utile e mi piace anche. Non saprei dirti riguardo le tempistiche prima di raggiungere un accordo sul nuovo design, e funzionalità. Credo che per un periodo si potranno sviluppare parallelamente, e intanto andare online come da #35 .

Già ripartendo con l'https, le discussioni qui e un sito operativo, credo che potremo anche dedicarci a sistemarlo in tempo reale (categorie, tipi di post etc) in maniera piu semplice ed efficace per tutti.

**Reply by dennisangemi** [reply]
> Per permettere agli utenti di rimanere in contatto facciamo un form per la mailing-list?

Edit (per un attimo ho confuso mailing list e newsletter) : Perchè vuoi fare un form? In mailing list si entra con [link google](https://groups.google.com/g/opendatasicilia), no?

> Mi sembra la soluzione piu pulita, semplice e sicura?

Quindi posso eliminare il login dalla navbar e magari inserirlo nel footer?

**Reply by gabacode** [reply]
> Perchè vuoi fare un form? In mailing list si entra con [link google](https://groups.google.com/g/opendatasicilia), no?

Mi sa che mi cono confuso anch'io, credevo ci fosse la mailing list di ODS ma mi sono confuso con quella di OnData!
> Quindi posso eliminare il login dalla navbar e magari inserirlo nel footer?

Per me si, se usiamo il link "celato" tipo wp-admin.php?key=0SE9T8ALSFK e' meglio non metterlo proprio.

**Reply by dennisangemi** [reply]
ok allora temporaneamente elimino qualsiasi riferimento al login. In caso lo inseriamo in footer

**Comment by dennisangemi** [comment]
Scusate se ritorno su questa discussion ma non mi è chiara una cosa di cui si è discusso anche nella #40 : per il nuovo sito i commenti verranno gestiti con disqus? aggiudicato?

**Comment by dennisangemi** [comment]
So che eravamo d'accordo a non integrare i commenti ma date un'occhiata qua https://twitter.com/squidfunk/status/1482650810299437056?s=19
